lab-scott/lib/fs-module.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = exports = {};
+
+const fs = require('fs');
+
+exports.hexArr = [];
+exports.reads = function(cb){
+  let fileArr = ['one.txt', 'two.txt', 'three.txt'];
+  for (let i=0; i < fileArr.length; i++){
+    fs.readFile(`${__dirname}/../data/${fileArr[i]}`, function(err, data) {
+      if(err) throw err;
+      exports.hexArr.push(data.toString('hex',0,8));
+      // console.log(`${fileArr[i]}: ${data.toString('hex',0,8)}`);
```

Comment:
**TODO FOR LOST POINTS**

Take out corpse code. 

Also, I realize you're trying to escape callback hell, but this for loop actually isn't going to solve the problem. Your last test (the last ``it`` block) fails because the returned array is always inconsistent because the async-ness is not being handled properly. 

I.e., I get different hex values coming back in different orders in the array because the way this module is written, there is no way you are guaranteeing that they are pushed into the array in the correct order you want them. 

The old way you had it with nested callbacks actually worked just fine, so revert to that and resubmit.